dcs/task.py

```diff
@@ -1077,6 +1082,39 @@ def __eq__(self, other) -> bool:
         return False
 
 
+class CarpetBombing(Task):
+    Id = "CarpetBombing"
+
+    class AttackType(Enum):
+        Carpet = "Carpet",
+
+        def __eq__(self, other: Any) -> bool:
+            if isinstance(other, str):
+                return self.value == other
+            return self == other
+
+    def __init__(self, altitude: int = 2000, altitude_enabled: bool = False,
+                 attack_qty: int = 1, attack_qty_limit: bool = False,
+                 attack_type: AttackType = AttackType.Carpet, carpet_length: int = 500,
+                 expend: Expend = Expend.Auto, group_attack: bool = False,
```

Comment:
Group attack doesn't appear to be an option in the ME.

(I'm looking at Perform Task -> Carpet bombing, and have looked at B-52, B-1, and B-17, all of which have the same options)

Reply:
If you can't explain their behavior we shouldn't expose them. We might be doing more harm than good by exposing them now. We don't want a DCS update to expose latent bugs in callers that were setting a previously ignored property.